vivbin

```diff
@@ -9,14 +8,26 @@ import cProfile
 import argparse
 import threading
 import traceback
+import logging
 
+logging.basicConfig(format='%(asctime)s:%(levelname)s:%(name)s: %(message)s')
+logger = logging.getLogger()
+logger.setLevel(logging.DEBUG)  # any messages called during import will be printed
```

Comment:
I think you can just pass `level=logging.DEBUG` to `logginb.basicConfig` to configure the root logger in one statement

```suggestion
logger.basicConfig(level=logging.DEBUG, format='%(asctime)s:%(levelname)s:%(name)s: %(message)s')
```